packages/engine/src/HedronEngine/HedronEngine.ts

```diff
@@ -26,6 +29,11 @@ export class HedronEngine {
     this.onFrameEnd = params?.onFrameEnd
   }
 
+  public registerPlugin(plugin: IPlugin) {
+    // plugins isn't actually used anywhere yet, but I assume it will be at some point
+    this.plugins.push(plugin)
```

Comment:
Not something we need now, but related to a comment above, we'll probably want to fire some event so that the desktop can update it's own `plugins` state, in case a plugin is registered after initial load of the app. Really don't think we need it anytime soon though!